src/app/store/effects/app.effects.ts

```diff
@@ -37,7 +37,7 @@ export class AppEffects {
     ofType(appActions.LOGOUT),
     map(() => {
       // TODO: shouldn't need to clear cookies here
-      this.cookieService.removeAll();
+      // this.cookieService.removeAll();
```

Comment:
Please do not change logout code unless you figured out a better way of fixing this.